pom.xml

```diff
@@ -197,6 +197,7 @@
     <maven-antrun.version>1.8</maven-antrun.version>
     <commons-crypto.version>1.1.0</commons-crypto.version>
     <commons-cli.version>1.2</commons-cli.version>
+    <bouncycastle.version>1.52</bouncycastle.version>
```

Comment:
Although this is a test case, this looks like too old because it's released on Mar, 2015. Do we need to align with to Hadoop 3.3.1? Can we use the latest one `1.68`?
- https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15on

Reply:
You're right - will change to `1.68`

Reply:
Actually Hadoop is reverting the change in https://issues.apache.org/jira/browse/HADOOP-17563 so we'll need to wait a bit. cc @steveloughran .

Reply:
Wonder why hadoop doesn't shade bouncycastle?

Reply:
Thank you for the pointer, @sunchao . It seems that they found CNFE with HBase due to the incompatibility across that library.

Reply:
bouncycastle was unshaded in https://issues.apache.org/jira/browse/HADOOP-15832. Reason is: "Leave bouncycastle unshaded because it's signed with a special Oracle certificate so it can be a custom JCE security provider"

Reply:
To be clear, bouncycastle was also unshaded in Hadoop 3.2.2 which Spark currently use, but since https://issues.apache.org/jira/browse/YARN-8448 it becomes a runtime dependency for RM and therefore required by Spark tests (e.g., `MiniYARNCluster`).

Reply:
we rolled back the latest bouncy castle update as some internal spark builds with some extra modules were failing as the asm JAR couldn't handle the changes. Tried to fiddle with the spark settings there to use an updated asm JAR and gave up